packages/flutter/lib/src/rendering/debug_overflow_indicator.dart

```diff
@@ -198,16 +198,23 @@ mixin DebugOverflowIndicatorMixin on RenderObject {
     return regions;
   }
 
-  void _reportOverflow(RelativeRect overflow, String overflowHints) {
-    overflowHints ??= 'The edge of the $runtimeType that is '
-      'overflowing has been marked in the rendering with a yellow and black '
-      'striped pattern. This is usually caused by the contents being too big '
-      'for the $runtimeType.\n'
-      'This is considered an error condition because it indicates that there '
-      'is content that cannot be seen. If the content is legitimately bigger '
-      'than the available space, consider clipping it with a ClipRect widget '
-      'before putting it in the $runtimeType, or using a scrollable '
-      'container, like a ListView.';
+  void _reportOverflow(RelativeRect overflow, List<DiagnosticsNode> overflowHints) {
+    overflowHints ??= <DiagnosticsNode>[];
+    if (overflowHints.isEmpty) {
+      overflowHints.add(ErrorHint(
+        'The edge of the $runtimeType that is '
+        'overflowing has been marked in the rendering with a yellow and black '
+        'striped pattern. This is usually caused by the contents being too big '
+        'for the $runtimeType.'
```

Comment:
I'd consider this part as an ErrorDescription.

Reply:
done